storey/dtypes.py

```diff
@@ -52,6 +53,23 @@ def __eq__(self, other):
     def __str__(self):
         return f'Event(id={self.id}, key={self.key}, time={self.time}, body={self.body})'
 
+    def copy(self, body=None, key=None, time=None, id=None, headers=None, method=None, path='/', content_type=None, awaitable_result=None,
+             deep_copy=False) -> 'Event':
```

Comment:
Shouldn't it be `-> Event`? Shouldn't `path=None` here?

Reply:
Should be `path=None` indeed, took defaults from constructor.
Can't do `-> Event`, Python 3.7 doesn't support referencing class type inside class.

Reply:
Ah ok, I see now that using a string works as well.